csharp/ql/test/query-tests/Security Features/CWE-079/XSSRazorPages/gen_files.py

```diff
@@ -0,0 +1,46 @@
+# A script for generating code from .cshtml files, mimicking the output of the C# compiler with an option that is not available from the codeql test runner.
+
+import sys
+import os
+
+work_dir = os.path.dirname(sys.argv[0])
```

Comment:
```suggestion
work_dir = os.path.abspath(os.path.dirname(sys.argv[0]))
```

Reply:
With this change it was possible for me to invoke the script by `python3 gen_files.py`.